### ISSUE TITLE
[a11y design review]: Vitals domain

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#55625]
## Auditor(s)

Sara Smith

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [ ] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [ ] WCAG 2.1 AA
  - [ ] WCAG 2.1 AAA
  - [ ] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [ ] High Contrast Mode
- [ ] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav)
  - [ ] Annotations for Tab order, reflow and reading order (if needed)
- [ ] Accessible Name Annotations
  - [ ] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [ ] Text alternative for complex images such as charts graphs maps etc.
- [ ] Language (ease of reading)
- [ ] HTML Tags
  - [ ] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [ ] [Heading order](https://accessibility.18f.gov/headings/)
   - [ ] Annotations (if needed)
  - [ ] Semantic Consideration for paragraphs ordered lists numbering
  - [ ] [Tables (heading rows)](https://design.va.gov/components/table)
- [ ] Size of clickable areas

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

## Prototype Link

- Vitals list view [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/4E480FBD-C72A-435B-8F8B-CB8031A673AA) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/5ACFEE0F-2FFC-42AB-BFD9-C1E69FBACE4E)
  - Details page variations:
    - Vitals/Blood pressure [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/BB8932F5-531E-4CB2-B66E-24B194BAD904) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/6210A759-096D-4821-BA3B-EB79DCC5164A)
    - Vitals/Height [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/367F815C-AEAD-4911-8C4B-AC5BED825C58) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/1F057337-F44E-4D98-9DB0-43477F5E5ADD)
    - Vitals/Pain level [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/19939DB4-47DF-4C38-95A6-31157D604B4F) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/159A2770-70EA-4324-BC78-D716A22B581D)
    - Vitals/Pulse Rate [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/7633BF05-6505-480C-B5AE-57FD34F70FFE) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/85424C3F-1BBC-4D69-8B8E-89335D2200A9)
    - Vitals/Respiration [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/DA2D624B-C3F1-437C-9341-6F76E2698D4C) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/9E12E49B-E6EB-4451-8E2D-C3FC3340D8D1)
    - Vitals/Temperature [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/34076153-8072-4D31-81E4-17EC4207D2FD) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/67E54149-D1FA-4BFB-985E-976B889B323B)
    - Vitals/Weight [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/1558EE8F-3C1D-4974-9061-586F45B7FFA2) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/0C6A591B-364A-4662-A56C-D99BC2298D3F)
    - Vitals/PDF (download view) [Page 1](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/4AD1388E-9E8E-4C5A-9902-9342D9CC8F2C) | [Page 2](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/800FB665-6802-4FAA-891A-BCFAADEF008B)

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- If no research has been conducted with assistive technology users, it should be a priority

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

**Comment by sara-amanda** [comment]
This was consolidated for the Medical Records Midpoint Review.

**Comment by RLHecht** [comment]
Thanks, Sara. Closing.
